Adjust jest docs (#1128)
MetaMask/metamask-docs: snaps/reference/jest.md

});
```

### `onHomePage`

> The `onHomePage` function can be used to request the home page of the snap. It
> takes no arguments, and returns a promise that resolves to the response from the
> [`onHomePage`](entry-points#onhomepage)
> function.

Reviewer: ```suggestion
Requests the home page of the Snap. It
takes no arguments, and returns a promise that resolves to the response from the
[`onHomePage`](entry-points#onhomepage)
entry point.
```

You can pass the following options when [configuring `@metamask/snaps-jest`](../how-to/test-a-snap.md#2-configure-metamasksnaps-jest).
All options are optional.

> ### `browser`

Reviewer 2: This was removed
